include/mapnik/json/feature_generator_grammar.hpp

```diff
@@ -63,6 +63,18 @@ struct end_container<mapnik::feature_impl const>
     }
 };
 
+#if BOOST_VERSION >= 106900
```

Comment:
Should this be `>= 107000`?  As it stands, building with Boost 1.69.0 fails for me.

Reply:
Hmm.. I just checked and it works for me on linux+clang. Could you share your setup, pls?